elf_reader.go

```diff
@@ -1258,7 +1258,7 @@ func getProgType(sectionName string) (ProgramType, AttachType, uint32, string) {
 		{"cgroup/getsockopt", CGroupSockopt, AttachCGroupGetsockopt, 0},
 		{"cgroup/setsockopt", CGroupSockopt, AttachCGroupSetsockopt, 0},
 		{"struct_ops+", StructOps, AttachNone, 0},
-		{"sk_lookup/", SkLookup, AttachSkLookup, 0},
+		{"sk_lookup", SkLookup, AttachSkLookup, 0},
```

Comment:
FWIW we aren't compatible with libbpf here, since we treat all of these names as a prefix. Might make sense to pull this into a separate PR and discuss there.

Reply:
Yeah, I've started noticing that as well. I'll take it out for now.

Reply:
We should probably track this as a larger work item, changing only sk_lookup doesn't make things any more consistent.

Reply:
Created this: https://github.com/cilium/ebpf/issues/1162.